lib/web/apiserver.go

```diff
@@ -443,6 +443,9 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*APIHandler, error) {
 	// GET /webapi/sites/:site/desktopplayback/:sid?access_token=<bearer_token>
 	h.GET("/webapi/sites/:site/desktopplayback/:sid", h.WithAuth(h.desktopPlaybackHandle))
 
+	// GET a Connection Diagnostics by its name
+	h.GET("/webapi/sites/:site/diagnostics/connections/:name", h.WithClusterAuth(h.getConnectionDiagnostic))
```

Comment:
when i lookced at this to hook it up with UI, i didn't know what `name` meant, and looking it up it said `connection name`, that is the connection ID right? could we maybe rename to `connectionId`? or just `id`?

Reply:
Yeah, not the best name I would say
It was like this because the actual field is `Name`

Changing to `connectionid` :+1: